lib/items/twitter.dart

```diff
@@ -12,19 +17,29 @@ class TwitterItem extends MultaccItem {
 
   toMap() => {'at': username, 'id': userId};
 
-  get humanReadableValue => (username == null || username.trim() == '') ? '' : '@$username';
+  get humanReadableValue => '@${username ?? ''}';
```

Comment:
Should not just display '@' if username is unavailable

```suggestion
  get humanReadableValue => (username ?? '') == '' ? '' : '@$username';
```